### PR TITLE
Reporting Unit Tests with Manual Optimization

### PR DESCRIPTION
**Pull request by dbeylkin**
## Prerequisites

- [x] I have read the [Contributing Guidelines](../blob/main/CONTRIBUTING.md).
- [x] I have read the [Code of Conduct](../blob/main/CODE_OF_CONDUCT.md).
- [x] If this is a non-trivial change, I have already opened an accompanying Issue.
- [x] If applicable, I have included documentation updates alongside my code changes.

## Description

Remove network optimization from reporting unit tests. Instead, manually generate proposed topology.

Note: manually generated topology comes from the optimized output; however, readjust_sectors_post_opt is not called, so some reporting values change.

With this change, the unit tests pass for both Xpress v8.13.0 and v8.14.2

## Test Plan

Ran reporting tests with both Xpress versions.

### CODE REVIEW

File: docs/runbook/Quick_Start.md [hunk 1]
```diff
@@ -61,6 +61,10 @@ Run the tests to check if the package is correctly installed.
 python3 -m unittest discover terragraph_planner -b
 ```
 
+> Using different versions of 3rd party dependencies than the ones specified
```
Comment (wuweicaifb) [inline docs/runbook/Quick_Start.md]: than -> from?

File: docs/runbook/Quick_Start.md [hunk 2]
```diff
@@ -61,6 +61,10 @@ Run the tests to check if the package is correctly installed.
...
+in the requirements file can lead to test failure; this does **not** mean the
```
Comment (wuweicaifb) [inline docs/runbook/Quick_Start.md]: not -> NOT?

Comment (dbeylkin) [reply docs/runbook/Quick_Start.md]: You prefer that to bold? Or both?

Comment (wuweicaifb) [reply docs/runbook/Quick_Start.md]: I prefer both.

**Review by wuweicaifb — APPROVED** [review]

**Pull request merged**